lib/cookie_policy_consent/blocs/cookie_policy_consent_bloc.dart

```diff
@@ -12,22 +14,34 @@ class CookiePolicyConsentBloc
   CookiePolicyConsentBloc(
     this.cookiePolicyConsent,
   ) : super(CookiePolicyConsentInitial()) {
-    on<CookiePolicyConsentEvent>((event, emit) async {
-      if (event is VerifyCookiePolicyConsent) {
-        emit(VerifyingCookieConsent());
+    on<CookiePolicyConsentEvent>(
+      (event, emit) async {
+        if (event is VerifyCookiePolicyConsent) {
+          emit(VerifyingCookieConsent());
 
-        final hasAcceptedCookiePolicyConsent =
-            await cookiePolicyConsent.hasAcceptedCookiePolicy();
+          final hasAcceptedCookiePolicyConsent =
+              await cookiePolicyConsent.hasAcceptedCookiePolicy();
 
-        if (hasAcceptedCookiePolicyConsent) {
+          if (hasAcceptedCookiePolicyConsent) {
+            emit(CookiePolicyConsentAccepted());
+          } else {
+            emit(CookiePolicyConsentRejected());
+          }
+        } else if (event is AcceptCookiePolicyConsent) {
+          cookiePolicyConsent.acceptCookiePolicy();
           emit(CookiePolicyConsentAccepted());
-        } else {
-          emit(CookiePolicyConsentRejected());
         }
-      } else if (event is AcceptCookiePolicyConsent) {
-        cookiePolicyConsent.acceptCookiePolicy();
-        emit(CookiePolicyConsentAccepted());
-      }
-    });
+      },
+      transformer: debounceSequential(const Duration(milliseconds: 100)),
```

Comment:
Can we have a smaller delay?
```suggestion
      transformer: debounceSequential(const Duration(milliseconds: 50)),
```